lib/elbas/aws/autoscale_group.rb

```diff
@@ -38,6 +39,13 @@ def query_autoscale_group_by_name(name)
             .first
         end
 
+        def launch_template_specification
+          aws_counterpart.launch_template ||
+            aws_counterpart.mixed_instances_policy.launch_template
+            .launch_template_specification
+        rescue NoMethodError
+          raise Elbas::Errors::NoLaunchTemplate
```

Comment:
We need to decide whether this method raises or the `launch_template` method, I don't think it should be both. Either this method should return `nil` here and leave the raise to L21, or we should remove L21 and raise here instead. What do you think?